gulp/tasks/doc.js

```diff
@@ -93,7 +94,7 @@ gulp.task('copy-pattern-library', function() {
 
 gulp.task('webpack', function() {
     return gulp.src('')
-        .pipe(webpack(webpackConfig))
+        .pipe(webpackStream(webpackConfig, webpack))
```

Comment:
Have to do this because `webpack-stream` installs a 1.x version of webpack by default